### PR TITLE
Add Spanish translation for routing.md

### PR DESCRIPTION
**Pull request by alemohamad**
Added Spanish translation for routing.md.

### CODE REVIEW

File: docs/basics/routing.es.md [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+Para configurar la estrategia de recopilación de Body en una ruta individual, usa el parámetro `body`.
+
+```swift
+// Recopila la transmisión de Body (hasta 1mb de tamaño) antes de llamar a esta ruta.
+app.on(.POST, "listings", body: .collect(maxSize: "1mb")) { req in
+    // Administra la petición. 
+}
+```
+
+Si se pasa el `maxSize` al realizar el `collect` (la recopilación), se sobreescribirá el valor predeterminado de la aplicación para esa ruta. Para utilizar el valor por defecto de la aplicación, omita el argumento `maxSize`.
```
Comment (jacostaf10) [inline docs/basics/routing.es.md:265]:
```suggestion
Si se proporciona el `maxSize` (tamaño máximo) al realizar el `collect` (la recopilación), se sobrescribirá el valor predeterminado de la aplicación para esa ruta. Para utilizar el valor por defecto de la aplicación, omita el argumento `maxSize`.
```

File: docs/basics/routing.es.md [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+app.get("hello", ":name") { req in
+	...
+}.description("says hello")
+```
+
+## Grupos de Ruta
+
+La agrupación de rutas permite crear un conjunto de rutas con un prefijo o un middleware específico. La agrupación admite una sintaxis basada en builders y closures.
+
+Todos los métodos de agrupación devuelven un `RouteBuilder`, lo que significa que se pueden mezclar, combinar y anidar infinitamente sus grupos con otros métodos de creación de rutas.
```
Comment (jacostaf10) [inline docs/basics/routing.es.md:327]:
```suggestion
Todos los métodos de agrupación devuelven un `RouteBuilder`, permitiendo mezclar, combinar y anidar infinitamente sus grupos con otros métodos de creación de rutas.
```

File: docs/basics/routing.es.md [hunk 3]
```diff
@@ -0,0 +1,436 @@
...
+    ...
+}
+// GET /users/:id
+users.get(":id") { req in
+    let id = req.parameters.get("id")!
+    ...
+}
+```
+
+Cualquier componente de ruta que puedas pasar como `get` o `post` puede pasarse dentro de un `grouped`. También hay una sintaxis alternativa, basada en sintaxis de closure.
```
Comment (jacostaf10) [inline docs/basics/routing.es.md:350]:
```suggestion
Cualquier componente de ruta que puedas pasar como `get` o `post` puede pasarse dentro de un `grouped`. También hay una sintaxis alternativa basada en closures.
```

**Review by jacostaf10 — COMMENTED** [review]
I proposed some changes, it's up to you to decide which fit best.

**Comment by alemohamad** [comment]
Thanks for the thorough review @jacostaf10 !!

File: docs/basics/routing.es.md [hunk 4]
```diff
@@ -0,0 +1,436 @@
...
++--------+----------------+
+| POST   | /todos         |
++--------+----------------+
+| DELETE | /todos/:todoID |
++--------+----------------+
+```
+
+### Metadata
+
+Todos los registro de métodos de rutas retornan la ruta creada (`Route`). Esto te permite agregar metadaatos al diccionario `userInfo` de la ruta. Hay algunos métodos predeterminados disponibles, como agregar una descripción a la ruta.
```
Comment (jacostaf10) [inline docs/basics/routing.es.md:315]:
```suggestion
Todos los registro de métodos de rutas retornan la ruta creada (`Route`). Esto te permite agregar metadatos al diccionario `userInfo` de la ruta. Hay algunos métodos predeterminados disponibles, como agregar una descripción a la ruta.
```

Comment (jacostaf10) [reply docs/basics/routing.es.md:315]: I missed that one on the previous check.

**Review by jacostaf10 — APPROVED** [review]
As soon as the misspelling is fixed, it is ready to go.

**Review by 0xTim — APPROVED** [review]

**Pull request merged**